### DISCUSSION TITLE
Is it possible to create a service account programmatically?

### DISCUSSION THREAD

**Discussion by shumybest** [minio/minio#14352] [Q&A] [answered]
Hello, I am facing an issue to manage the object access from mobile devices. 

I'd like to assign different appkey/secret (which is permanent) for each type of mobile device, the service account functionality meets my requirement. But I haven't found any of this API from sdk.

Is there any way to list/create/delete service account programmatically?

Thank you in advance.

**Comment by harshavardhana** [comment, marked as answer]
Yes https://pkg.go.dev/github.com/minio/madmin-go

**Reply by shumybest** [reply]
Great! Thank you very much.
